Makefile

```diff
@@ -0,0 +1,17 @@
+
+REGISTRY=gcr.io/ossf-malware-analysis
+
+.PHONY: all all_docker analysis_docker
+
+all: all_docker
+
+all_docker:
```

Comment:
Does this need a dependency on analysis_docker?

Reply:
I was just using it as a default 'all'. `all_docker` also builds analysis docker image.

Reply:
Ah right, the bash script also builds it. 

Can you just add a comment here explaining the distinction between the two cases? i.e. build_docker.sh builds everything, including analysis.

One other option is to make the build_docker.sh take in arguments, and the Makefile can just pass arguments there to build the relevant images.

Reply:
Since I'm hoping to merge `build_docker.sh` into this script, I'll probably just leave a comment here for now